dev/io.openliberty.microprofile.telemetry.1.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -13,36 +13,37 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>io.openliberty.microprofile.telemetry</groupId>
-    <artifactId>tck.runner.tck</artifactId>
-    <version>3.0</version>
-    <name>MicroProfile Telemetry 1.0 TCK Runner TCK Module</name>
-    
-    <repositories>
+    <!--<repositories>
         <repository>
             <id>eclipse-staging</id>
             <url>https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1632/</url>
         </repository>
-    </repositories>
+    </repositories>-->
```

Comment:
I think we can just delete this, it was only present when we needed to reference the staged artifacts.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/24211